CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 ## [Unreleased]
 
+# 1.12.1
+### Added
+- `--output` flag indicates which package type you want to create deb\rpm\any fpm supported output.
+    Notice that the default is DEBIAN if you dont give the flag. 
```

Comment:
please link to the issue here (and not to ZenHub as it's not open). Look at line 25 for reference.

Reply:
I would change the text to 
```
### Changed
- Debify now receives the flag `--output` as input to indicate the file type that it should package (e.g `rpm`). If this 
  flag is not given, the default value is `deb`.
  <link to github issue>

Reply:
Done